bundle/schema/docs/bundle_descriptions.json

```diff
@@ -36,7 +51,7 @@
         }
       }
     },
-    "targets": {
+    "environments": {
```

Comment:
This is intentional; we want to nudge folks to using "targets".

Reply:
This would mean a small regression for existing users of `environments`. Should we remove the dsecriptions all together?

FYI, I am fine with either.

Reply:
Removing it or deprecating it is an incentive to update it.

Btw, there is a `deprecated` property in JSON schema that could be used (if easy to do): https://json-schema.org/draft/2019-09/json-schema-validation#rfc.section.9.3